ikabot/helpers/botComm.py

```diff
@@ -46,36 +46,36 @@ def sendToBot(s, msg, Token=False):
 	"""
 	if Token is False:
 		msg = 'pid:{}\n{}\n{}'.format(os.getpid(), config.infoUser, msg)
-	sessionData = s.getSessionData()
+	sessionData = session.getSessionData()
 	try:
-		ikabot.web.sesion.normal_get('https://api.telegram.org/bot{}/sendMessage'.format(sessionData['telegram']['botToken']), params={'chat_id': sessionData['telegram']['chatId'], 'text': msg})
+		ikabot.web.session.normal_get('https://api.telegram.org/bot{}/sendMessage'.format(sessionData['telegram']['botToken']), params={'chat_id': sessionData['telegram']['chatId'], 'text': msg})
```

Comment:
session.normal_get, we have import there already

Reply:
sorry not sure what you mean

Reply:
right, it is not possible because we have `import ikabot.web.session` and we also have `session` (type Session) as parameter